app/src/examples/RuntimeTests/ReanimatedRuntimeTestsRunner/types.ts

```diff
@@ -1,7 +1,7 @@
 import { Component, Dispatch, MutableRefObject, ReactNode, SetStateAction } from 'react';
 import { AnimatedStyle, StyleProps } from 'react-native-reanimated';
 
-type CallTrucker = {
+export type CallTrucker = {
```

Comment:
Shouldn't it be named `CallTracker` or is that name reserved? 🚚

Reply:
Thank you, it is a real word, so my spell checker was fine with it 😆

Reply:
I'll take one more step forward and assume we are not going to need to use word "truck" at all, and add the following to `cspell.json`

<img width="380" alt="image" src="https://github.com/software-mansion/react-native-reanimated/assets/56199675/7905e0bf-ecb6-4063-a329-cc18047c627f">